src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -37,6 +37,7 @@ internal sealed class HttpConnection : IDisposable
         private readonly byte[] _writeBuffer;
         private int _writeOffset;
 
+        private Task<int> _initialRead;
```

Comment:
Can we call this something like _readAhead?  That seems more intuitive to me.  The point here isn't really so much that it's the initial read, as that it's a read issued in advance of the next request.